grafana-plugin/src/pages/integration/Integration.tsx

```diff
@@ -697,6 +692,17 @@ class Integration extends React.Component<IntegrationProps, IntegrationState> {
       })
       .finally(() => this.setState({ isLoading: false }));
   }
+
+  async loadContactPointsMaybe(id: AlertReceiveChannel['id']) {
+    const { alertReceiveChannelStore } = this.props.store;
+
+    if (IntegrationHelper.isGrafanaAlerting(alertReceiveChannelStore.items[id])) {
+      // this will be delayed and not awaitable so that we don't delay the whole page load
+      return await alertReceiveChannelStore.updateConnectedContactPoints(id).catch(noop);
```

Comment:
`.catch(noop);` I think this needs to be removed, because we don't call contact points endpoint if integration is non-GrafanaAlerting